freqtrade/pairlist/VolumePairList.py

```diff
@@ -69,7 +70,26 @@ def _gen_pair_whitelist(self, base_currency: str, key: str) -> List[str]:
         # check length so that we make sure that '/' is actually in the string
         tickers = [v for k, v in tickers.items()
                    if len(k.split('/')) == 2 and k.split('/')[1] == base_currency]
-
         sorted_tickers = sorted(tickers, reverse=True, key=lambda t: t[key])
-        pairs = [s['symbol'] for s in sorted_tickers]
+        # Validate whitelist to only have active market pairs
+        valid_pairs = self._validate_whitelist([s['symbol'] for s in sorted_tickers])
+        valid_tickers = [t for t in sorted_tickers if t["symbol"] in valid_pairs]
+
+        if self._freqtrade.strategy.stoploss is not None and self._precision_filter:
+
+            logger.debug(f"Markets: {list(self._markets)}")
+            stop_prices = [self._freqtrade.get_target_bid(t["symbol"], t)
+                           * (1 + self._freqtrade.strategy.stoploss) for t in valid_tickers]
```

Comment:
I would use `(1 - abs(self._freqtrade.strategy.stoploss)` ...
that would have it aligned to the "real" stoploss calculation in `persistence.py`.